contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -338,18 +336,8 @@ func (k *kubeScheduler) doSchedule(task *podtask.T, err error) (string, error) {
 			return "", fmt.Errorf("task.offer assignment must be idempotent, task %+v: offer %+v", task, offer)
 		}
 
-		// write resource limits into the pod spec which is transferred to the executor. From here
-		// on we can expect that the pod spec of a task has proper limits for CPU and memory.
-		// TODO(sttts): For a later separation of the kubelet and the executor also patch the pod on the apiserver
-		if unlimitedCPU := mresource.LimitPodCPU(&task.Pod, k.defaultContainerCPULimit); unlimitedCPU {
-			log.Warningf("Pod %s/%s without cpu limits is admitted %.2f cpu shares", task.Pod.Namespace, task.Pod.Name, mresource.PodCPULimit(&task.Pod))
-		}
-		if unlimitedMem := mresource.LimitPodMem(&task.Pod, k.defaultContainerMemLimit); unlimitedMem {
-			log.Warningf("Pod %s/%s without memory limits is admitted %.2f MB", task.Pod.Namespace, task.Pod.Name, mresource.PodMemLimit(&task.Pod))
-		}
-
 		task.Offer = offer
-		task.FillFromDetails(details)
+		k.api.algorithm().Procurement()(task, details) // TODO(jdef) why is nothing checking the error returned here?
```

Comment:
Thanks for surfacing this - I forgot about it!

On Thu, Sep 3, 2015 at 3:21 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In contrib/mesos/pkg/scheduler/plugin.go
> <https://github.com/kubernetes/kubernetes/pull/13421#discussion_r38618878>
> :
>
> >  		task.Offer = offer
> > -		task.FillFromDetails(details)
> > +		k.api.algorithm().Procurement()(task, details) // TODO(jdef) why is nothing checking the error returned here?
>
> How about a ticket if it's important?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13421/files#r38618878>.
>